apps/spruce/lint-staged.config.js

```diff
@@ -2,5 +2,5 @@ import baseConfig from "@evg-ui/lint-staged";
 
 export default {
   ...baseConfig,
-  "*.{ts,tsx}": () => ["yarn check-types", "yarn check-schema-and-codegen"], // For TypeScript files, run tsc, and gql schema check
```

Comment:
why drop check-types?

Reply:
It's already applied to all TS/TSX files in the base config!